behave/features/steps/general.py

```diff
@@ -54,6 +55,9 @@ def click_element(context, css_selector):
 def set_element_value(context, css_selector, element_value):
     html_element = context.browser.find_element_by_css_selector(css_selector)
     html_element.clear()
+    if css_selector == '#first_name':
+        date_stamp = date.today().strftime('%Y-%m-%d')
+        element_value = element_value+'_'+date_stamp
```

Comment:
It would be cleaner if you create a new function to set the element value to today's date and revert the amends to this function
e.g. `def set_element_value_to_date_today(context, css_selector)`

Reply:
addressed as commented !!!!